lib/ramble/ramble/cmd/workspace.py

```diff
@@ -137,8 +134,7 @@ def workspace_activate(args):
         env_mods = ramble.workspace.shell.deactivate()
 
     # Activate new workspace
-    workspace_path_dir = ramble.workspace.get_workspace_path()
-    active_workspace = ramble.workspace.Workspace(workspace_path_dir)
+    active_workspace = ramble.workspace.Workspace(workspace_path)
```

Comment:
I think there should be an override context right above this.

Reply:
lib/ramble/ramble/cmd/workspace.py was reverted to the exact form it had before PR#36. So if this change needs to be made, maybe it should be made as part of a separate PR?